pageserver/src/basebackup.rs

```diff
@@ -222,6 +222,8 @@ where
     async fn send_tarball(mut self) -> anyhow::Result<()> {
         // TODO include checksum
 
+        let on_demand_slru_download = self.timeline.get_lazy_slru_download() && !self.full_backup;
```

Comment:
```suggestion
        let lazy_slru_download = self.timeline.get_lazy_slru_download() && !self.full_backup;
```
Let's use the same terminology everywhere